data/fields/turning_circle.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "turning_circle",
+    "type": "combo",
+    "label": "Type"
```

Comment:
“Shape” would dissuade users from polluting the `turning_circle=*` key with values representing orthogonal aspects of a turning circle.